packages/admin/README.md

```diff
@@ -126,14 +126,15 @@ const contract = {
 await client.proposeUpgrade({ newImplementation }, contract);
 ```
 
-Alternatively, you can add any contract explicitly by using the `addContract` method, and setting network, address, name, and ABI. The same method can be used to update the contract's name or ABI.
+Alternatively, you can add any contract explicitly by using the `addContract` method, and setting network, address, name, natspec and ABI. The same method can be used to update the contract's name or ABI.
```

Comment:
Just double checking, aren't we allowing them to update natspec? (if so we should also mention it in the last sentence here)

Reply:
oh actually I just checked and we do not as of now

Reply:
@BjMrq could you please capture updating natspec in a separate issue? Otherwise lgtm